packages/hash/api/src/model/knowledge/orgMembership.model.ts

```diff
@@ -102,22 +95,19 @@ export default class extends EntityModel {
       {
         all: [
           {
-            equal: [
-              { path: ["leftEntity", "uuid"] },
-              { parameter: this.getEntityUuid() },
-            ],
+            equal: [{ path: ["uuid"] }, { parameter: this.getEntityUuid() }],
           },
           {
             equal: [
-              { path: ["leftEntity", "ownedById"] },
+              { path: ["ownedById"] },
               { parameter: this.getOwnedById() },
             ],
           },
           {
             equal: [
-              { path: ["type", "versionedUri"] },
+              { path: ["rightEntity", "type", "versionedUri"] },
               {
-                parameter: SYSTEM_TYPES.linkEntityType.ofOrg.getSchema().$id,
+                parameter: SYSTEM_TYPES.entityType.org.getSchema().$id,
               },
             ],
           },
```

Comment:
If this class where to extend `LinkEntityModel`, the pre-fetched `this.rightEntityModel` could be used here instead.

Reply:
Handled by 9fa54ada81cf0e59578d49550d729a9a1dcc90d2 (Thanks Alfie!)